src/components/IOUConfirmationList.js

```diff
@@ -102,21 +112,66 @@ const defaultProps = {
     comment: '',
     network: {},
     myPersonalDetails: {},
+    iouType: CONST.IOU.IOU_TYPE.REQUEST,
 };
 
 class IOUConfirmationList extends Component {
     constructor(props) {
         super(props);
 
-        this.toggleOption = this.toggleOption.bind(this);
-
         const formattedParticipants = _.map(this.getParticipantsWithAmount(this.props.participants), participant => ({
             ...participant, selected: true,
         }));
 
+        // Add the button options to payment menu
+        const confirmationButtonOptions = [];
+        let defaultButtonOption = {
+            text: this.props.translate(this.props.hasMultipleParticipants ? 'iou.split' : 'iou.request', {
+                amount: this.props.numberFormat(
+                    this.props.iouAmount,
+                    {style: 'currency', currency: this.props.iou.selectedCurrencyCode},
+                ),
+            }),
+        };
+        if (this.props.iouType === CONST.IOU.IOU_TYPE.SEND && this.props.participants.length === 1) {
+            // Add the Expensify Wallet option if available and make it the first option
+            if (this.props.localCurrencyCode === CONST.CURRENCY.USD || Permissions.canUsePayWithExpensify(this.props.betas)) {
```

Comment:
Just to make sure I'm understanding, people with USD who are _not_ on the beta can send with the wallet? 

Also we may want to check for the wallet beta here too (otherwise you can't see your wallet balance)